server/base_client.py

```diff
@@ -34,9 +36,10 @@ def __init__(self, conn, server=None):
         self.__status = None
         self.data_buffer = DataBuffer()
         self.data_parser = DataParser()
+        self.addr = "(%s:%s)" % (self.conn.getpeername()[0], self.conn.getpeername()[1])
```

Comment:
Guys, it's not an issue, but maybe it's better to follow one style. E.g. I see mix of format and % sting builders.

Reply:
fixed